test/helpers/expectBNIsWithinDelta.js

```diff
@@ -0,0 +1,8 @@
+const expectBNIsWithinDelta = (actualBN, expected, allowableDelta) => {
```

Comment:
In `master` we're using [openzeppelin-test-helpers](https://github.com/OpenZeppelin/openzeppelin-test-helpers), which will let you use `bignumber.closeTo(expected, delta)` :) (see [here](https://github.com/ZeppelinSolutions/chai-bn/commit/994e3280de8630b12a2fbd1240ce12f9597eca0e))

Reply:
working on it.